src/uwtools/tests/drivers/test_standalonedriver.py

```diff
@@ -0,0 +1,246 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+Tests for uwtools.drivers.standalonedriver module.
```

Comment:
```suggestion
Tests for uwtools.drivers.driver module.
```

Reply:
If both base and sub driver classes continue to live in the same `uwtools.drivers.driver` module, the test modules should probably be merged together, too, to try to keep the one-to-one mapping we (mostly) have between test modules and modules-under-test. That might call for test classes like `ConcreteDriver` and `ConcreteAssets`, if those are the names we use, so that they can co-exist in the test module.

Reply:
Also fair - I had observed that but stubbornly kept them separated. They should have their own test classes and go together.